docs/UserGuide.md

```diff
@@ -629,34 +629,27 @@ Format: `list`
 
 ### Finding patients: `find`
 
-Finds patients whose names contain any of the given keywords.
-
-Format: `find [KEYWORD]… [n/NAME]… [p/PHONE]… [e/EMAIL]… [a/ADDRESS]… [t/TAG]… [c/CONDITION]… [d/TASK_DESCRIPTION]… [m/MEDICATION]… [r/REMARK]…`
+Finds patients whose parameters contain the given keywords.
```

Comment:
Perhaps it may be better to use more second-person language (I had updated it for conditions, tags, and other miscellaneous commands). 

```suggestion
You can find specific patients using the `find` command.
```